tests/system/test_fim/test_synchronization/test_synchronization.py

```diff
@@ -0,0 +1,154 @@
+"""
+copyright: Copyright (C) 2015-2021, Wazuh Inc.
+           Created by Wazuh, Inc. <info@wazuh.com>.
+           This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
+type: system
+brief: Check that when FIM is activated, and the agent is running, the agent and manager are synchronization when
+a change is performed in a monitored folder.
+tier: 1
+modules:
+    - fim
+components:
+    - manager
+    - agent
+path: tests/system/test_fim/test_fim_synchronization/test_fim_synchronization.py
+daemons:
+    - wazuh-syscheckd
+os_platform:
+    - linux
+os_version:
+    - Arch Linux
+    - Amazon Linux 2
+    - Amazon Linux 1
+    - CentOS 8
+    - CentOS 7
+    - CentOS 6
+    - Ubuntu Focal
+    - Ubuntu Bionic
+    - Ubuntu Xenial
+    - Ubuntu Trusty
+    - Debian Buster
+    - Debian Stretch
+    - Debian Jessie
+    - Debian Wheezy
+    - Red Hat 8
+    - Red Hat 7
+    - Red Hat 6
+references:
+    - https://github.com/wazuh/wazuh-qa/issues/2434
+tags:
+    - fim_synchronization
+"""
+
+import os
+import json
+import pytest
+from time import sleep
+
+
+from wazuh_testing.tools.monitoring import HostMonitor
+from wazuh_testing.tools.system import HostManager
+from test_fim import create_folder_file, clean_logs, query_db
+
+
+# Hosts
+testinfra_hosts = ["wazuh-manager", "wazuh-agent1"]
```

Comment:
When do you use it?

Reply:
Done in commit [080c1](https://github.com/wazuh/wazuh-qa/pull/2536/commits/080c12ab527c54c7dbed0a7b22af7c2cf4c668e0)